ale_linters/html/htmlhint.vim

```diff
@@ -1,7 +1,7 @@
 " Author: KabbAmine <amine.kabb@gmail.com>, deathmaz <00maz1987@gmail.com>, diartyz <diartyz@gmail.com>
 " Description: HTMLHint for checking html files
 
-call ale#Set('html_htmlhint_options', '--format=unix')
+call ale#Set('html_htmlhint_options', '--format=unix --config .htmlhintrc')
```

Comment:
The command could be executed from any directory, and the `.htmlhintrc` file could lie in some parent directory.

Instead of doing this, I recommend setting `--config` automatically if an `.htmlhintrc` file is found in the directory or in a parent directory. `autoload/ale/fixers/rubocop.vim` has a pretty good example for this kind of thing. Make sure to add a test for this. Use `git add -f` on a dummy dotfile to add it to repository.